src/test/java/com/will/jpapractice/domain/OrderPersistenceTest.java

```diff
@@ -0,0 +1,133 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+
+@DataJpaTest
+@Slf4j
+public class OrderPersistenceTest {
+    @Autowired
+    EntityManagerFactory emf;
+
+    EntityManager entityManager;
+
+    @BeforeEach
+    void setUp() {
+        entityManager = emf.createEntityManager();
+    }
+
+    @Test
+    void 주문_생성_후_회원에_주문_등록() {
+        // Given
+        Member member = createMember();
+
+        // When
+        Order newOrder = new Order(member);
+        Order memberOrder = member.getOrders().get(0);
+
+        // Then
+        assertThat(memberOrder).usingRecursiveComparison().isEqualTo(newOrder);
+    }
+
+    @Test
+    void 주문상품_생성_후_주문에_주문상품_등록() {
+        // Given
+        Member member = createMember();
+        Order order = new Order(member);
+        Item newItem = createItem();
+
+        // When
+        OrderItem newOrderItem = new OrderItem();
+        newOrderItem.setOrder(order);
+
+        // Then
+        OrderItem orderItem = order.getOrderItems().get(0);
+        assertThat(orderItem).usingRecursiveComparison().isEqualTo(newOrderItem);
+    }
+
+    @Test
+    void 주문_삭제() {
+        // Given
+        Member member = createMember();
+        Order order = new Order(member);
+        order.setUuid(UUID.randomUUID().toString());
+
+        Item newItem = createItem();
+
+        OrderItem newOrderItem = new OrderItem();
+        newOrderItem.setOrder(order);
+        newOrderItem.setItem(newItem);
+
+        entityManager.persist(order);
+
+        // When
+        entityManager.remove(order);
+
+        // Then
+        Order selectedOrder = entityManager.find(Order.class, order.getUuid());
+        assertThat(selectedOrder).isNull();
+    }
+
+    @Test
+    void 연관관계_테스트() {
+        // Given
+        EntityTransaction tx = entityManager.getTransaction();
+
+        tx.begin();
+
+        Member member = createMember();
+
+        entityManager.persist(member);
+
+        Order order = new Order();
+        order.setUuid(UUID.randomUUID().toString());
+        order.setCreatedAt(LocalDateTime.now());
+        order.setOrderStatus(OPENED);
+        order.setMemo("---");
+        order.setMember(member);
+
+        entityManager.persist(order);
+
+        tx.commit();
+
+        // When
+        entityManager.clear();
+        Order selectedOrder = entityManager.find(Order.class, order.getUuid());
+        Member selectedMember = entityManager.find(Member.class, member.getId());
+
+        // Then
+        assertThat(selectedOrder.getMember().getNickName()).isEqualTo(member.getNickName());
+        assertThat(selectedMember.getOrders()).hasSizeGreaterThan(0);
```

Comment:
```suggestion
        assertThat(selectedMember.getOrders()).hasSize(1);
```
이렇게 하는게 더 명확하지 않을까요?

Reply:
위에서 Order 하나를 만들었기 때문에 hasSize(1)이 더 명확하겠네요!